### PR TITLE
feat: allocation tweaks

### PR DESCRIPTION
**Pull request by jaekwon**
This is part of an investigation and research into memory allocation issues in test1.gno.land, the post to "Describe in your own words..." which had almost a thousand comments.

Each comment rendered took initially some 200k bytes of allocation, and toward the end took 800k bytes of allocation per comment, probably from the cumulative += of the rendered body, as well as inspection into the AVL tree's inner nodes during iteration. (And this was calculated by injecting a debug statement during iteration that printed the allocation figures per comment render). 

While we could consider temporarily allowing no allocation limit for queries, this is not a real solution against gno.land realm page renders that eat up memory. 

Also, this is related to #266, as solving #266 could resolve this issue easily, but that's still not the ultimate general solution, as some dapp functions will inevitably encounter the memory limit.

Probably the ultimate solution is to implement pagination in rendering. For test3.gno.land, we will implement pagination for boards.

### CODE REVIEW

File: gnoland/genesis/genesis_txs.txt [hunk 1]
```diff
@@ -5,91 +5,17 @@
...
+{"msg":[{"@type":"/vm.m_call","caller":"g1us8428u2a5satrlxzagqqa5m6vmuze025anjlj","send":"","pkg_path":"gno.land/r/boards","func":"CreateBoard","args":["testboard"]}],"fee":{"gas_wanted":"2000000","gas_fee":"1gnot"},"signatures":[{"pub_key":{"@type":"/tm.PubKeySecp256k1","value":"A+FhNtsXHjLfSJk1lB8FbiL4mGPjc50Kt81J7EKDnJ2y"},"signature":"vzlSxEFh5jOkaSdv3rsV91v/OJKEF2qSuoCpri1u5tRWq62T7xr3KHRCF5qFnn4aQX/yE8g8f/Y//WPOCUGhJw=="}],"memo":""}
```
Comment (jaekwon) [inline gnoland/genesis/genesis_txs.txt:17]: find-in-page for more references to "testboard"

Comment (jaekwon) [reply gnoland/genesis/genesis_txs.txt:17]: for example, the transaction below also replaces "gnolang" with "testboard"

File: gnoland/genesis/genesis_txs.txt [hunk 2]
```diff
@@ -5,91 +5,17 @@
...
 {"msg":[{"@type":"/bank.MsgSend","from_address":"g1us8428u2a5satrlxzagqqa5m6vmuze025anjlj","to_address":"g15gdm49ktawvkrl88jadqpucng37yxutucuwaef","amount":"100000gnot"}],"fee":{"gas_wanted":"50000","gas_fee":"1gnot"},"signatures":[{"pub_key":{"@type":"/tm.PubKeySecp256k1","value":"AmG6kzznyo1uNqWPAYU6wDpsmzQKDaEOrVRaZ08vOyX0"},"signature":"AsNIHs4pX8FshGwWsIXMjAYi4NG7uc+DA75wVq986NZXVhTbSNJJZPdbDboKe7mSSblneP8MFCuF+TICxd83xg=="}],"memo":""}
-{"msg":[{"@type":"/bank.MsgSend","from_address":"g14vhcdsyf83ngsrrqc92kmw8q9xakqjm0v8448t","to_address":"g1xha96z6yztjxh85040t3jdwaela53cp0d40ujz","amount":"1gnot"}],"fee":{"gas_wanted":"200000","gas_fee":"1gnot"},"signatures":[{"pub_key":{"@type":"/tm.PubKeySecp256k1","value":"AgiTyNXOXskkkxKv2DXaDxFtabuTPHlltbYwCOBpHL3w"},"signature":"VzhOTGiwR2oiTSbrybBNcyHJJ4sdMP8xRbQsf+6eqaNebDVKeTLnrxzgAlMrgRvQvA0rJD54m6/MjJHJVBEMfg=="}],"memo":""}
```
Comment (jaekwon) [inline gnoland/genesis/genesis_txs.txt:8]: removed bad transactions.

**Review by moul — APPROVED** [review]

**Comment by r3v4s** [comment]
I was debugging some bug on gno-vm, and found this pr
Just curious, can allocation limit(tx 0.5gb, query 1.5gb) be increased after proper discussion? or is it like constant value?

**Pull request merged**